core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreaker.java

```diff
@@ -73,4 +73,13 @@ static CircuitBreaker ofDefaultName() {
      * Decides whether a request should be sent or failed depending on the current circuit state.
      */
     boolean canRequest();
+
+    /**
+     * Enters the specified {@link CircuitState}. Note that even if the {@link CircuitBreaker}
+     * is already in the specified {@link CircuitState}, timeouts such as
+     * {@link CircuitBreakerConfig#circuitOpenWindow()} and {@link CircuitBreakerConfig#trialRequestInterval()}
```

Comment:
`CircuitBreakerConfig` is not a public class unfortunately.

Reply:
@jrhee17, you marked this as resolved but it seems not? :thinking:

Reply:
Sorry 😅  First this happened to me, but seems like `git push` updated the remote ref but didn't actually push changes to the server for some reason. Sorry about the confusion 🙏 

<img width="1617" alt="Screen Shot 2022-01-24 at 3 09 50 PM" src="https://user-images.githubusercontent.com/8510579/150731086-b5f17fdb-6226-4631-8f46-3c073d251352.png">